debug.go

```diff
@@ -1,9 +1,12 @@
 package fuse
 
 import (
+  "os"
 	"runtime"
 )
 
+var IsRunningUnderTravis = len(os.Getenv("TRAVIS")) > 0
```

Comment:
This is used to disable certain tests. Instead of `IsRunningUnderTravis` I recommend stating the feature that prevents tests or code from running.

If you would run fuse under certain conditions it would apparently fail for a subset of the methods.